Task08/CreateData.java

```diff
@@ -0,0 +1,49 @@
+package test.HW8;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+/**
+ * Class creates list of products
+ */
+public class CreateData {
+    ArrayList<ArrayList<SomeSubject>> listOfEntity = new ArrayList<>();
+    InputData input = new InputData();
+
+    /**
+     * Method create list of lists.
+     * The first element of any list is the TYPE of product.
+     * The user chooses to expand list or not (Yes or Not)
+     *
+     * @throws IOException
+     */
+    public void createData() throws IOException {
```

Comment:
OMG, double infinite loop! As it was discussed in class avoid infinite loop. Even one level.